code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Dispenser.dm

```diff
@@ -275,7 +279,8 @@
 	specific_heat = 1.181
 
 /datum/reagent/iron/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
-	M.add_chemical_effect(CE_BLOODRESTORE, 8 * removed)
+	if (alien != IS_SKRELL)
```

Comment:
IS_SKRELL is a bitflag and should be treated as such. While I can't think of any mechanic or race that makes you a hybrid, you should always respect currently implemented systems.

So this should look like ```alien & IS_SKRELL``` and the other one ```!(alien & IS_SKRELL)```

Reply:
I won't pretend to understand what a bitflag is at this point in time, but I can say that I think I understood what you wanted changed and I think I changed it correctly.